pom.xml

```diff
@@ -150,6 +154,34 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <!-- Produce a jar with a platfrom-specific classifier. -->
```

Comment:
We actually generate and attach the JAR with the native library using `maven-antrun-plugin` above. I guess we don't need it.